### ISSUE TITLE
How to make the component to work

### ISSUE THREAD

**Issue by dirodriguezm** [jjdeu/animated-header-component#1]
Hi, I'm sorry about the dumb question but I have trouble making this work in my project. I even tried copying my project files to this demo but no luck.

Can you provide some installation instructions ?

Great work by the way :)

**Comment by jjdeu** [comment]
Hi, no such things as dumb questions :-)
Have you tried cloning the whole project and getting it to work?
What exactly isn't working? Are there any console warnings / errors?

If you got my repository working on your own machine the easiest way would be to follow the example. So add a components directory to you /src. Copy the animated-header folder to the components directory. Add the AnimatedHeaderComponentClass to the declarations in app.module.ts

Then in the page where you want to use the component follow the example in home.html and home.scss
Basically you leave everything intact except the part where I have put in the ion-list. There you can put your own content (cards, lists, etc).
Make sure that the picture you want to use at the top is in the assets/imgs directory
The image is used in animated-header.component.scss (bottom of the file).

Just let me know if it's still not working.

Cheers, JJ
